src/processgraph/commons.js

```diff
@@ -3,17 +3,42 @@ const Utils = require('../utils');
 
 module.exports = class ProcessCommons {
 
-	static reduceInCallback(node, reducer) {
+	static reduceInCallback(node, reducer, ...args) {
 		var dc = node.getData("data");
 		var func = data => data.reduce(reducer);
 		if (node.getProcessGraph().isSimpleReducer() || dc.isImageCollection()) {
-			dc.imageCollection(func);
+			dc.imageCollection(func, ...args);
+			// rename bands according to the GEE convention
```

Comment:
Is this renaming backed by the process spec? The band dimension should be removed, I think. We should have a way to work with it on Googles side. So maybe it's better to rename the bands on Googles side to a fixed name that indicates us there's no band left? Not sure myself, but needs discussion...

Reply:
Currently we are always reducing over time, since this is the only way to do it for image collections. However, if we convert them to arrays we could resolve this and also reduce along the band dimension. For now I would just rename them back to their original name (e.g. 'VV_mean' -> 'VV'). Then the user can access the bands as intended, without any knowledge about the GEE naming convention.

Reply:
That sounds reasonable.